titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/producer/JooqJobActivityProducerConnectorComponent.java

```diff
@@ -0,0 +1,54 @@
+package com.netflix.titus.ext.jooqflyway.jobactivity.producer;
+
+import javax.sql.DataSource;
+
+import com.netflix.titus.ext.jooqflyway.jobactivity.RDSSSLSocketFactory;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.jooq.DSLContext;
+import org.jooq.SQLDialect;
+import org.jooq.impl.DSL;
+import org.postgresql.PGProperty;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+@Configuration
+public class JooqJobActivityProducerConnectorComponent {
```

Comment:
What is the difference between this database connection and the other one?

Reply:
It was complaining about having two data sources with one configuration, I tried separating it out but after talking to runtime, marking one as primary with correct qualifiers seems to work and is the recommended way